napari/layers/base/_tests/test_base_key_bindings.py

```diff
@@ -0,0 +1,11 @@
+from napari.layers.base import _base_key_bindings as key_bindings
+from napari.layers.points import Points
+
+
+def test_hold_to_pan_zoom(layer):
+    data = [[1, 3], [8, 4], [10, 10], [15, 4]]
+    layer = Points(data, size=1)
+
+    layer.mode = 'transform'
+    # need to go through the generator
+    _ = list(key_bindings.hold_to_pan_zoom(layer))
```

Comment:
Is this just testing that this code does not error? Is there any return value or state we could inspect to check an meaningful value change?

Reply:
This is unchanged, just moved from a different file. Not sure what it was supposed to test, but it returns `[None]`... I can confirm that the behaviour (holding space to pan_zoom) works as intended. Need to check what exactly is going on here.

Reply:
Ok, I updated it to actually check that the layer mode is affected.